### ISSUE TITLE
Enabling markSpecialLinks causes lock icons in tabs in edit

### ISSUE THREAD

**Issue by jaroel** [plone/plonetheme.barceloneta#147]
When markSpecialLinks is enabled in @@theming-controlpanel -> 'Advanced settings' -> 'Theme base' then lock icons appear next to tabs when the site is served over https.

![screen shot 2017-09-29 at 16 43 04](https://user-images.githubusercontent.com/70700/31021168-532ebc76-a535-11e7-94e2-947cd8c0482b.png)
![screen shot 2017-09-29 at 16 43 53](https://user-images.githubusercontent.com/70700/31021212-75b44b44-a535-11e7-8489-469c7350d128.png)

**Comment by jaroel** [comment]
ps, the SSL cert warning does not cause this.

**Comment by jaroel** [comment]
Of course, when I file a ticket I can find all the other tickets reporting the same.
https://github.com/plone/mockup/issues/707

**Comment by vincentfretin** [comment]
see https://github.com/plone/Products.CMFPlone/issues/1966 too
